### DISCUSSION TITLE
Robotics rework, new roboticist job.

### DISCUSSION THREAD

**Discussion by nymhole** [space-wizards/space-station-14#31724] [Feature Request]
since law board have been introduced, i say we add the ability to customize borgs laws.
There should be a roboticist job, with a new roboticist access. this is because i think we should give them more power over the borgs they create.

**Law Boards:** law boards should be made by roboticists, these law boards when made should have default laws already inscribed into them, you should be able to choose which default law set you are printing.

**Law Editing Console:** There should be a law editing console, which accepts law boards that a roboticist can make. by default, the console should not be able to edit above the default lawset's lowest ranking law. (example law 3: do not hurt yourself) this can be used to hold a borg to a specific department when not actively following an order given by a crew member, or whatever other obligations are given to them by their default lawset are. (example law 4: stay in medical and heal crewmembers).

**Forensics:** The log probe when used on the robotics console should be able to see who added what laws at what time.

**Subverting Borgs** this would also change how subverting borgs works. emagging a borg should only disable the connection with the robotics console the same way it does now, emagging a robotics console should allow the console to edit all laws on a law board. this would allow for a more specific and low-profile way of completing tasks as a syndicate. (example law 0: prevent Fenfir Vatia from boarding the evacuation shuttle by any means necessary.) 

**Ion Laws** ion laws should affect the way that the borg chassis interprets laws, not the law boards themselves. this can include adding extraneous laws, changing the order of laws, removing the ability to state some laws, removing the ability to read some laws, changing words within the laws read, or replacing laws when there is a law board inserted, or just a complete override of the law board adding in the laws superficially. 

**Borg Construction:** the physical location of the law board should be under the brain, you should have to put in a law board before a brain. the brain should be able to be placed within a chassis without a law board, but you cannot add a law board to a borg with a brain inside of it.

**New Devices:** A handheld law editor and syndicate equivalent for nukies to use. the RD (or [Chief of Robotics](https://forum.spacestation14.com/t/robotics-rework-roboticist-job-added/11676/2?u=nymhole)) would start with it, and nukie agent would start with it or would be able to buy it. a thief may be able to start with the syndicate version of it too, but not an emag in the same bundle, maybe a nukie thief bundle? or it fits somewhere else, similar to the gorlex hypospray and the hypospray.

**Comment by lzk228** [comment]
probably requires a design doc

**Comment by DandiIion** [comment]
Law editing would be nice, especially for emergencies. Borgs have been forced pacifists during zombies and nukies for way too long now. 

Random roboticists though should really not be allowed to edit the laws keep that to RD/command. ie responsible people (in theory)

"used to hold a borg to a specific department" That sounds awful borg players can already do their job without weird laws about it. I can imagine being forced to do science shite all shift already (I hate science shite). =/

**Reply by nymhole** [reply]
[this is a repost](https://forum.spacestation14.com/t/robotics-rework-roboticist-job-added/11676/6?u=nymhole)
i see where you’re coming from and this does make sense. i was only giving an example and i dont think that a borg would be held to science, however that is a possibillity with this proposal.
giving the ability to random roboticists who can be irresponsible could be an issue, but with a warden-adjacent role watching roboticists, maybe adding some surveillance device(s) for this role, and forensics to investigate the situation thereafter, i think it could be a good change (i think that being a borg can be boring at times, and it would be fun with a little more direction). assistant engineers can singuloose and they only have the CE to watch them. roboticists would require additional science and borg time.
in ss13 there are law racks that hold laws for all silicons, right? i think adding this is simply TOO powerful for antagonists, without more additions/changes to the AI and AI lawset first.

**Reply by LittleNorthStar** [reply]
Perhaps similar to the criminal records computer and cargo console you could have the console itself announced any changes to Cyborg laws unless the console has been tampered with, this would give a nice balance of roboticists being able to control laws while also allowing other Scientists and the RD supervision of their actions

**Reply by nymhole** [reply]
> Perhaps similar to the criminal records computer and cargo console you could have the console itself announced any changes to Cyborg laws unless the console has been tampered with, this would give a nice balance of roboticists being able to control laws while also allowing other Scientists and the RD supervision of their actions

i really like this idea.

**Comment by nymhole** [comment]
[this is a repost](https://forum.spacestation14.com/t/robotics-rework-roboticist-job-added/11676/16?u=nymhole)
this idea needs to be re-thought. cyborg laws should update with AI laws, and all cyborgs should report to the AI.
